src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1299,6 +1299,10 @@
     <Resource Include="UI\Images\profile_normal.png" />
   </ItemGroup>
   <ItemGroup>
+    <Resource Include="UI\Images\PackageManager\close_16_16.png" />
+    <Resource Include="UI\Images\PackageManager\down_16_16.png" />
+    <Resource Include="UI\Images\PackageManager\folder_16_16.png" />
+    <Resource Include="UI\Images\PackageManager\up_16_16.png" />
```

Comment:
@sm6srw maybe we should store our icons like this too

Reply:
Yes, I will take a look.